src/status_im/ui/screens/communities/invite.cljs

```diff
@@ -51,6 +51,7 @@
             contacts                    (map (fn [{:keys [public-key] :as contact}]
                                                (assoc contact :active (contains? selected public-key)))
                                              contacts-data)
+            is-there-contacts?          (seq contacts)
```

Comment:
(when-not (seq contacts)` is pretty readable, so no need in additional let

Reply:
Done